ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ip-dhcp-snooping.j2

```diff
@@ -0,0 +1,15 @@
+{% if ip_dhcp_snooping.enabled is arista.avd.defined(true) %}
+!
+ip dhcp snooping
+| VLAN |
+| ---  |
+{%   for vlan in ip_dhcp_snooping.vlans | arista.avd.natural_sort %}
```

Comment:
Maybe we could add a second column to this table with the state of DHCP snooping. Current documentation provide list of cli in addition to the CLI include.

Reply:
@titom73 can you provide an example either in one of the other documentation jinja templates or likewise?

Reply:
@burnyd you can look at `vlans.j2` in the documentation folder. In general, we use a table with at least 2 columns with this approach. The CLI part is done with the `include` in the second part of your file:


```jinja2
## VLANs Summary

| VLAN ID | Name |
| ------- | ---- |
{%     for vlan in vlans | arista.avd.natural_sort %}
{%         set row_name = vlans[vlan].name | arista.avd.default('-') %}
| {{ vlan }} | {{ row_name }} |
{%     endfor %}
```